code/renaming_scripts/vertices.py

```diff
@@ -1,25 +1,33 @@
-import sys
+import argparse
 import nibabel as nib
 
-if len(sys.argv) < 2:
-    print("Usage: python vertices.py <path_to_gii_file>")
-    sys.exit(1)
+def main():
+    parser = argparse.ArgumentParser(
+        description="Calculate the number of vertices in a GIFTI (.gii) file."
+    )
+    parser.add_argument(
+        "gii_file",
+        help="Path to the GIFTI (.gii) file"
+    )
 
-input_file = sys.argv[1]
+    args = parser.parse_args()
 
-gii = nib.load(input_file)
-arrays = gii.get_arrays_from_intent('NIFTI_INTENT_POINTSET')
+    gii = nib.load(args.gii_file)
+    arrays = gii.get_arrays_from_intent('NIFTI_INTENT_POINTSET')
 
-if not arrays:
-    # No pointset array found, print 0 or some error code
-    print("0")
-    sys.exit(0)
+    if not arrays:
```

Comment:
I think you can do 2 different options here:

1. Raise an error - I don't remember all of the errors Python has built-in and which ones would be good.
2. Still do a system exit.

I would normally go with 1, but since you are ultimately writing this values to a CSV file, I would just make sure this is caught downstream!